schedule-additional-tests

```diff
@@ -29,10 +29,13 @@ for x in `grep ADDITIONAL_TESTS=  $CONFIG_MAP | grep "SCRAM_ARCH=$ARCHITECTURE;"
     rm -f ${pfile}
   fi
   if [ "$x" = "profiling" ] ; then
-    PROFILING_WORKFLOWS=$(echo $(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||'), | tr ' ' ','| tr ',' '\n' | grep '^[0-9]' | sort | uniq | tr '\n' ',' | sed 's|,*$||')
-    prof="pp"
-    echo "PROFILE=$prof" >> ${pfile}
```

Comment:
@gartung you are missing this in new file

Reply:
you can have something like
```
echo "PROFILE=$prof" >> ${pfile}
echo "EVENTS=20" >> ${pfile}
```

in $pfile as these are common variables and only add  workflow number for $nfile

Reply:
I removed that parameter since it has no meaning. Both pp and mp profiles are run by the profiling scripts.

Reply:
ah ok